examples/advanced_tensorflow/server.py

```diff
@@ -26,8 +27,17 @@ def main() -> None:
         initial_parameters=fl.common.weights_to_parameters(model.get_weights()),
     )
 
-    # Start Flower server for four rounds of federated learning
-    fl.server.start_server("[::]:8080", config={"num_rounds": 4}, strategy=strategy)
+    # Start Flower SSL-enabled server for three rounds of federated learning
+    fl.server.start_server(
+        "0.0.0.0:8080",
```

Comment:
Shouldn't this align with the value set in `client.py`?
```suggestion
        "localhost:8080",
```

Reply:
The server should imo listen for all hosts. So I think 0.0.0.0 is better as otherwise only clients on the same machine would be able to connect.